raiden/network/proxies/token_network.py

```diff
@@ -387,7 +389,7 @@ def _new_netting_channel(
             block_identifier=encode_hex(receipt["blockHash"]),
         ).channel_identifier
 
-        return channel_identifier
+        return (channel_identifier, receipt["blockHash"], receipt["blockNumber"])
```

Comment:
I would have returned `return (channel_identifier, receipt)` to keep the number of return values low, but this is fine, too.